alpha/veneer/README.md

```diff
@@ -0,0 +1,17 @@
+
+## Veneers 
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions, giving operator authors a [declarative, deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/), but it can get complex, especially as the number of releases and dependencies scale.
+
+Enter the `veneer`.  The term is a bit of a misnomer, because it isn’t something pretending (or re-cladding) something, but it is instead a simpler transitional step to generating an FBC.
+
+The single characteristic that defines a veneer in this functional context is that when it is processed it will always yield a valid FBC. The tool that is used to process these veneers into the FBC product is not relevant. 
+
+There is great potential benefit to set expectations & establish consistent expectations by providing examples as part of `opm`, as well as a set of good use-cases to achieve it, representing "low-hanging fruit" to satisfy common use-cases.  
```

Comment:
WDYT of turning this into a bulleted list? These are really compelling reasons to use the veneers. Think the reasons would be clearer if they were broken up a bit. Maybe something like: 

The use of veneers provide the following benefits to <pipelines/teams/customers/partners>:

- They establish consistent expectations by providing examples as part of `opm`.
- They present a set of good use cases to achieve <noun>.
- They satisfy common use-cases, representing "low-hanging fruit".